lib/api/objectPutPart.js

```diff
@@ -177,6 +180,31 @@ function objectPutPart(authInfo, request, streamingV4Params, log,
                         cipherBundle, splitter);
                 });
         },
+        // If data backend is real AWS, no need to store part metadata
+        (destinationBucket, objectLocationConstraint, cipherBundle,
+        splitter, next) => {
+            if (config.backends.data === 'multiple') {
+                return multipleBackendGateway.uploadPart(
+                destinationBucket, objectLocationConstraint, objectKey,
+                uploadId, partNumber, log, (err, data) => {
+                    if (err) {
+                        log.error('error putting part to data backend', {
+                            error: err,
+                            method:
+                            'objectPutPart::multipleBackendGateway.uploadPart',
+                        });
+                        return next(err, destinationBucket);
+                    } else if (data) {
+                        // skip to end of waterfall
+                        return next(skipError, data);
+                    }
+                    return next(null, destinationBucket,
```

Comment:
please add comment that if backend doesn't handle mpu part, continue with putting steps